lib/core/utils/preload-cssom.js

```diff
@@ -1,310 +1,451 @@
+/* eslint no-use-before-define: 0 */
```

Comment:
It might be nice to include "why" we're disabling this rule here.

Reply:
added a note https://github.com/dequelabs/axe-core/pull/1364/files#diff-93d57bbe6c6d128e06d4fe309e06ff40R2